spec/features/user_logs_in_spec.rb

```diff
@@ -23,8 +23,9 @@
 
     expect(page).to have_content('Logged in successfully')
     expect(page).to have_content(user.username)
-    expect(page).to have_content('My Account')
     expect(page).to have_selector('li.avatar')
+    expect(page).to have_content('View Account')
+    expect(page).to have_content('Edit Account')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.